### PR TITLE
doc improvement: Add link to Pelion development tools

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:NCSDK-10733

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_pelion.rst [hunk 1]
```diff
@@ -37,6 +38,12 @@ Create Pelion developer account
 
 To use the Pelion Device Management library, you need a developer account on `Pelion Device Management Portal`_.
 
+Download Pelion development tools
+=================================
+
+The `Pelion Development Tools`_ allow creation of update resources and manifest.
+It is also possible to run update campaigns directly from a command line.
```
Comment (greg-fer) [inline doc/nrf/ug_pelion.rst:45]:
```suggestion
The `Pelion manifest tool`_ simplifies the creation of update resources and manifests.
You can also use them to run update campaigns directly from command line.

To download and install the tool, see `Installing the manifest tool`_ in the Pelion repository.
```

File: doc/nrf/ug_pelion.rst [hunk 2]
```diff
@@ -62,6 +69,7 @@ Provision the device with update resources
 ==========================================
 
 For details about provisioning the device with update resources, see the `Device Management Update`_ guide in the Pelion documentation.
+The `Pelion Development Tools` can be used to simplify update process during development stage.
```
Comment (greg-fer) [inline doc/nrf/ug_pelion.rst:72]:
```suggestion
You can also use the `Pelion manifest tool`_ to run the update process during development stage.
See `Using the manifest tool`_ in the Pelion repository for details.
```

File: doc/nrf/links.txt [hunk 3]
```diff
@@ -98,6 +98,7 @@
 .. _`nRF Cloud JSON protocol schemas`: https://github.com/nRFCloud/application-protocols
 
 .. _`Pelion Device Management Client library reference example`: https://github.com/PelionIoT/mbed-cloud-client-example
+.. _`Pelion Development Tools`: https://github.com/PelionIoT/manifest-tool
```
Comment (greg-fer) [inline doc/nrf/links.txt:101]:
```suggestion
.. _`Pelion manifest tool`: https://github.com/PelionIoT/manifest-tool#device-management-manifest-cli-tool
.. _`Installing the manifest tool`: https://github.com/PelionIoT/manifest-tool#installing-the-manifest-tool
.. _`Using the manifest tool`: https://github.com/PelionIoT/manifest-tool#using-the-manifest-tool
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Comment by greg-fer** [comment]
@pdunaj , please add entry in the changelog.

**Comment by pdunaj** [comment]
The last force push is just a rebase.

**Comment by pdunaj** [comment]
It seems we are already referencing older version of pelion manifest tool from pelion_client app. I have added an explicit information that this manifest tool is for version 4.7 of update client. Newer manifest tool should still work (if used according to documentation). We could drop the information about the older version altogether but I think it would be safest to keep it as is for now. @greg-fer what do you think?

**Comment by greg-fer** [comment]
@pdunaj , I'd go for the safest option, given how close the release is, and I think you've chosen the safest one.

**Review by greg-fer — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

**Pull request merged**